client/Components/Hearder/Menu.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import image from '../../img/logo.jpg';
+
+
+/**
+ * menu component unathenticated users
+ * @constant
+ */
+const Menu = () => (
+  <div>
+    <nav>
+      <div className="nav-wrapper black">
+        <a href="#!" className="brand-logo"><img src={image} className="responsive-img" alt="" style={{ height: 25 }} />More Recipes</a>
```

Comment:
Why are you using an anchor tag instead of React Link?

Reply:
It was an oversight I will change it. Okay thanks.